src/main/scala/wdl4s/AstTools.scala

```diff
@@ -16,7 +17,9 @@ object AstTools {
     def findAstsWithTrail(name: String, trail: Seq[AstNode] = Seq.empty): Map[Ast, Seq[AstNode]] = {
       AstTools.findAstsWithTrail(astNode, name, trail)
     }
-    def findTerminals(): Seq[Terminal] = AstTools.findTerminals(astNode)
+    def findTerminalsWithTrail(terminalType: String, trail: Seq[AstNode] = Seq.empty): Map[Terminal, Seq[AstNode]] = {
```

Comment:
These aliases were originally just to minimize the impact of some changes I made a while back. Since this is a new one it might as well be fully and completely be in here instead of a passthrough.